### PR TITLE
Spanner: make system test table easier to read

### PR DESCRIPTION
**Pull request by chemelnucfin**
This is to make the table easier to read and make the new tests easier to write.

### CODE REVIEW

File: spanner/tests/system/test_system.py [hunk 1]
```diff
@@ -388,20 +388,23 @@ class TestSessionAPI(unittest.TestCase, _TestData):
...
-        ([3, None, 9], False, None, None, float('-inf'), 207, u'bat', None),
-        ([], False, None, None, float('nan'), 1207, u'owl', None),
-        ([], False, None, None, OTHER_NAN, 2000, u'virus', NANO_TIME),
+        ([]          , False, None  , None , 0.0    , None, None  , None),
+        ([1]         , True , BYTES1, DATE1, 0.0    , 19  , u'dog', SOME_TIME),
+        ([5, 10]     , True , BYTES1, None , 1.25   , 99  , u'cat', None),
+        ([]          , False, BYTES2, None , POS_INF, 107 , u'man', None),
+        ([3, None, 9], False, None  , None , NEG_INF, 207 , u'bat', None),
+        ([]          , False, None  , None , NAN1   , 1207, u'owl', None),
+        ([]          , False, None  , None , NAN2   , 2000, u'pig', NANO_TIME),
```
Comment (dhermes) [inline spanner/tests/system/test_system.py]: See https://www.python.org/dev/peps/pep-0008/#pet-peeves:

```
# More than one space around an assignment (or other) operator to align it with another.
# Yes:
x = 1
y = 2
long_variable = 3
# No:
x             = 1
y             = 2
long_variable = 3
```

**Comment by chemelnucfin** [comment]
I understand.  This was just to be easier to read the table and write tests.  I wanted to separate the PR to make sure if this was preferred or not.  

I have to somewhat disagree with the PEP-8 here because it's a table with rows and columns.

I can change the table back locally after I'm done writing tests.

Feel free to close this.

**Comment by dhermes** [comment]
> I have to somewhat disagree with the PEP-8 here because it's a table with rows and columns.

Everyone does have their own local opinions, but to avoid disagreement, we stick with an already standard document.

**Comment by tseaver** [comment]
> Feel free to close this.

I don't like the spacing, either.

**Pull request closed without merging**